Catalog/README.md

```diff
@@ -0,0 +1,52 @@
+# Low Code App Editor
+
+[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=SkylineCommunications_Low-Code-App-Extensions&metric=alert_status)](https://sonarcloud.io/summary/new_code?id=SkylineCommunications_Low-Code-App-Extensions)
```

Comment:
Does it make sense to show Quality Gate Status here? As customer I would expect everything that is available to me to have passed the quality gate.

Reply:
I don't mind myself. It's a tangible evidence so that might be good to show. I'm not yet considering though to make this the default, but today I'm not seeing this as a blocker.